src/coreclr/debug/createdump/createdumpmain.cpp

```diff
@@ -141,7 +140,11 @@ int createdump_main(const int argc, const char* argv[])
             }
             else if (strcmp(*argv, "--address") == 0)
             {
-                options.SignalAddress = (void*)atoll(*++argv);
+                options.SignalAddress = atoll(*++argv);
```

Comment:
Are we always using decimal when reading in? I would have assumed we use hex which can be parsed using `strtoll()`

Reply:
This is address (and the exception record address) are always formatted as decimal.